director/template/vars_file_arg.go

```diff
@@ -29,6 +39,18 @@ func (a *VarsFileArg) UnmarshalFlag(filePath string) error {
 		return bosherr.WrapErrorf(err, "Deserializing variables file '%s'", filePath)
 	}
 
+	if fileScope != "" {
+		scopedVars := map[interface{}]interface{}{}
```

Comment:
This seems necessary due to [typing in `find_op`](https://github.com/cppforlife/go-patch/blob/5097ab27332566f622afb1549a5423ae0012b93f/patch/find_op.go#L86) which conflicts with [`StaticVariables`](https://github.com/cloudfoundry/bosh-cli/blob/5646ea8481b15e3db2dbeb2ec1ddc4bf2189569e/director/template/static_vars.go#L3). Otherwise this sort of error happens...

    Finding variable 'iaas.subnet_id':
      Expected to find a map at path '/subnet_id' but found 'map[string]interface {}'

Not sure if this is the correct solution.